packages/tailwindcss/src/ast.ts

```diff
@@ -314,6 +323,52 @@ export function optimizeAst(ast: AstNode[], designSystem: DesignSystem) {
           usedKeyframeNames.add(keyframeName)
       }
 
+      // Create fallback values for usages of the `color-mix(…)` function that reference variables
+      // found in the theme config.
+      if (node.value.includes('color-mix(')) {
+        let ast = ValueParser.parse(node.value)
+
+        let didGenerateFallback = false
+        ValueParser.walk(ast, (node) => {
+          if (node.kind !== 'function' || node.value !== 'color-mix') return
+
+          ValueParser.walk(node.nodes, (node, { replaceWith }) => {
+            if (node.kind !== 'function' || node.value !== 'var') return
+            let firstChild = node.nodes[0]
+            if (!firstChild || firstChild.kind !== 'word') return
+
+            let inlinedColor = designSystem.theme.resolveValue(null, [firstChild.value as any])
+            if (!inlinedColor) return
+
+            didGenerateFallback = true
+            replaceWith({ kind: 'word', value: inlinedColor })
+          })
+
+          // Change the colorspace to `srgb` since the fallback values should not be represented as
+          // `oklab(…)` functions again as their support in Safari <16 is very limited.
+          let colorspace = node.nodes[2]
+          if (
+            (colorspace.kind === 'word' && colorspace.value === 'oklab') ||
+            colorspace.value === 'oklch' ||
+            colorspace.value === 'lab' ||
+            colorspace.value === 'lch'
+          ) {
```

Comment:
While likely fine, we are only checking the type of the colorspace.kind to be `word` for the `oklab` value, not for all the other values.

I think we want this instead:
```suggestion
          if (
            colorspace.kind === 'word' &&
            (colorspace.value === 'oklab' ||
              colorspace.value === 'oklch' ||
              colorspace.value === 'lab' ||
              colorspace.value === 'lch')
          ) {
```

Reply:
Oh yep huh